Source/CarthageKit/Xcode.swift

```diff
@@ -650,6 +650,10 @@ private func build(sdk: SDK, with buildArgs: BuildArguments, in workingDirectory
 								// setting for GCC as noted in
 								// https://developer.apple.com/library/content/qa/qa1964/_index.html.
 								"CLANG_ENABLE_CODE_COVERAGE=NO",
+								
+								// Disable injecting the debugger entitlement into archive builds
+								// https://download.developer.apple.com/Developer_Tools/Xcode_9.3_beta/Release_Notes_for_Xcode_9.3_beta.pdf
```

Comment:
This link doesn't actually work, so it's not of much use.

Reply:
I don't know what apple is doing that link it's what shows in the address bar when I view the patch notes with a logged int account. Since it might not be public until GM here is the section that explains this new build flag. 


>The debugger on macOS now requires the entitlement com.apple.security.get-task- allow to attach to apps built for macOS or for iOS, tvOS or watchOS apps built for Simulator. Xcode automatically injects this entitlement to your builds. The entitlement is stripped from apps distributed using the Organizer window.
>
>To disable Xcode injecting the entitlement, set CODE_SIGN_INJECT_BASE_ENTITLEMENTS to NO in the build settings for the target or app. (34638816)

Reply:
I think they issue you a temporary permission via a redirect when you click on a link from elsewhere.

Reply:
I hope they have a public link to some documentation ready as a 9.3 release gets closer.